polkadot/runtime/westend/constants/src/lib.rs

```diff
@@ -130,6 +130,8 @@ pub mod xcm {
 		const ROOT_INDEX: u32 = 0;
 		// The bodies corresponding to the Polkadot OpenGov Origins.
 		pub const FELLOWSHIP_ADMIN_INDEX: u32 = 1;
+		#[deprecated = "Will be removed after August 2024; Use `xcm::latest::BodyId::Treasury` \
```

Comment:
why not just remove it?

Reply:
to make sure none uses the same index for the time of deprecation